packages/core/src/shared/telemetry/spans.ts

```diff
@@ -195,6 +205,10 @@ export class TelemetrySpan<T extends MetricBase = MetricBase> {
      */
     public start(): this {
         this.#startTime = new globals.clock.Date()
+        if (this.#options.trackPerformance) {
+            this.#performance?.start()
```

Comment:
With other suggestion:
```suggestion
            (this.#performance ??= new PerformanceTracker(this.name)).start()
```